### PR TITLE
docs: readme: add plugins section to example

### PR DESCRIPTION
**Pull request by coderaiser**
Added `plugins` section to example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,10 @@ $ npm install --save-dev eslint eslint-plugin-n
         "n/prefer-global/url": ["error", "always"],
         "n/prefer-promises/dns": "error",
         "n/prefer-promises/fs": "error"
-    }
+    },
+    "plugins": [
+        "n"
+    ]
```
Comment (aladdin-add) [inline README.md:42]: it's not needed any more - it has been set in the recommended preset:
https://github.com/weiran-zsd/eslint-plugin-node/blob/a6a31bbd4fcf4bceff11d383f851cd9d9b7ce05b/lib/configs/recommended-script.js#L19

**Comment by coderaiser** [comment]
I have a question related to `plugins` section and `FlatConfig`. I have an error:

```js
coderaiser@localcmd:~/putout/packages/plugin-remove-debugger$ eslint lib

Oops! Something went wrong! :(

ESLint: 8.23.0

TypeError: Key "plugins": Cannot redefine plugin "n".
    at ObjectSchema.merge (/Users/coderaiser/putout/node_modules/eslint/lib/config/flat-config-schema.js:272:23)
```

Here is my `eslint.config.js`:

```js
const {recommended} = require('eslint-plugin-putout/config');

module.exports = [
    ...recommended, {
    },
];
```

And content of `recommended.js`:

```js
const putoutPlugin = require('..');
const nPlugin = require('eslint-plugin-n');

const {FlatCompat} = require('@eslint/eslintrc');

const compat = new FlatCompat({
    baseDirectory: __dirname,
});

const rmPlugins = (a) => {
    delete a.plugins;
    return a;
};


// ----> without removing each plugin `n` we have an error
const nPluginReady = compat.config(nPlugin.configs.recommended)//.map(rmPlugins);

const config = [{
    plugins: {
        putout: putoutPlugin,
    },
}];

module.exports.recommended = [
    ...nPluginReady,
    ...compat.config(putoutPlugin.configs.recommended),
    ...config,
];

module.exports.safe = [
    ...nPluginReady,
    ...compat.config(putoutPlugin.configs.safe),
    ...config,
];

module.exports.safeAlign = [
    ...nPluginReady,
    ...compat.config(putoutPlugin.configs['safe+align']),
    ...config,
];
```

`eslint-plugin-putout` [has a line](https://github.com/coderaiser/putout/blob/master/packages/eslint-plugin-putout/lib/index.js#L108-L110):

```js
{
    plugins: [
        'n',
    ],
}
```

But has no extends section.

So the question is: can I declare the same plugin couple times using `plugins` section? In `eslint-plugin-putout` it used as dependency, since `n` used in `rules` section (so it for frontend and backend). And in `config` path `recommended` is used, since it for `node.js`.

So to get thing works I'm removing `plugins` if want to use `n:recommended` with `putout` (or I should remove plugins from `eslint-plugin-putout`, but in this case nothing will work if user configuration don't use `n:recommended`, because it for frontend...).

I'm wandering is it a bug or I'm doing something wrong?

**Comment by aladdin-add** [comment]
seems it was an issue in the new experimental config, I've filed it here: https://github.com/eslint/eslint/issues/16277

please follow that issue for new update. :)

**Comment by coderaiser** [comment]
Nice :), thank you!

**Pull request closed without merging**